terraform/dotnet/ec2/default/main.tf

```
@@ -116,7 +116,11 @@ resource "null_resource" "main_service_setup" {
       sudo yum install -y wget
       sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
       sudo wget -O /etc/yum.repos.d/microsoft-prod.repo https://packages.microsoft.com/config/fedora/37/prod.repo
-      sudo dnf install -y dotnet-sdk-8.0
+      if [[ "${var.language_version}" == "8.0" ]]; then
+        sudo dnf install -y dotnet-sdk-8.0
+      else
+        sudo dnf install -y dotnet-sdk-6.0
+      fi
```

nit: Should use 8.0 as default
```suggestion
      if [[ "${var.language_version}" == "6.0" ]]; then
        sudo dnf install -y dotnet-sdk-6.0
      else
        sudo dnf install -y dotnet-sdk-8.0
      fi
```